src/test/java/com/sonyericsson/jenkins/plugins/bfa/BuildFailureScannerHudsonTest.java

```diff
@@ -325,6 +325,31 @@ public void testNoMultilineIndicationFound() throws Exception {
     }
 
     /**
+     * Tests that "no problem identified" message is not shown when no failure cause has been found.
```

Comment:
Too many negations here, sorry I am not English mother-tongue and couldn't think of anything even slightly more readable.